packages/scripts/download-repos.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+set -eu;
+
+# Clone an organization's repositories into a folder.
+# Example:
+#   packages/scripts/download-repos.sh ../repos
+
+if [ -z "$1" ] ; then
+  echo "Provide path to clone repos into e.g. ../repos"
+  exit 1
+fi
+
+BASEDIR=$(dirname "$0")
+
+DEBUG=scripts:* $BASEDIR/get-repos.js | \
+  while read -r gitRepoUrl gitRepoName; \
+  do \
+    echo "${gitRepoName} cloning..."; \
+    git clone $gitRepoUrl $1/$gitRepoName --quiet; \
```

Comment:
you probably want to do `--bare` or `--mirror` to include all branches. Test example: magazine repo.

see https://help.github.com/articles/duplicating-a-repository/